### DISCUSSION TITLE
ERGO and KASPA Dualmining HiveOs Nvidia AMD PLS make the Dream come True before the Merge !

### DISCUSSION THREAD

**Discussion by Skydiver911** [Lolliedieb/lolMiner-releases#1699] [General]
Before the Merge strikes in and we all get lost, pls make the dream come true, that we could mine ERGO and Dual Mining KASPA .

Many Miners must Dual Mine and LOLMINER is always for me the right Choice , meanwhile i dual mine ETH and KAS and its really great.

After Merge ERG would be for the most one the only decision because of its less Energy Costs , if we could dualmine KAS with it, it could

be the next Skyrocket for LOLMINER worldwide.

Best Regards,

Nicole

**Comment by Cybertiron** [comment]
Yes the potential is huge here for lolminer, I truly believe in them that they can do it.

**Comment by siiinaaa** [comment]
ye 
plz give us Ergo + kaspa on nvidia :)

**Comment by machac2018** [comment]
Wolud be great to have ERG+KAS in Lolminer currenlty using another miner but I prefer LoL

**Comment by tomekmak** [comment]
And + ZIL maybe as 3rd coin. That could be awesome
